modules/ossm-threescale-webassembly-module-backend-object.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-backend-object_{context}"]
+= The 3scale WebAssembly module backend object
+
+The `backend` top-level object specifies how to access the 3scale Service Management API for authorizing and reporting HTTP requests. This service is provided by the _Backend_ component of 3scale.
+
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshExtension
+metadata:
+  name: threescale-wasm-auth
+spec:
+  config:
+    ...
+    backend:
+      name: backend
+      upstream: <object_see_below_table>
```

Comment:
I might suggest avoiding using the replaceable name here to give instructions (i.e., "see below table"). Maybe just a callout like:

```
      upstream: <object> <1>
```

and then:

```
<1> See the following table.
```

Or if you wanted to give the below table a title (like `.backend object fields` or similar), you could refer to it by name like:

```
<1> See the "backend object fields" table.
```

Or maybe simply `<object>` with no callout is fine enough, since the table is immediately following.

Reply:
If you do change this, check the other instances of `<object_see_below_table>` as well, in case those can also be changed.